Infusionsoft/ContactGroupAssign.php

```diff
@@ -1,7 +1,24 @@
-<?php
-class Infusionsoft_ContactGroupAssign extends Infusionsoft_Generated_ContactGroupAssign{	
+<?php
+class Infusionsoft_ContactGroupAssign extends Infusionsoft_Generated_ContactGroupAssign{
+
     public function __construct($id = null, $app = null){
-    	parent::__construct($id, $app);    	    	
+        parent::__construct(null, $app);
+        if($id != null) {
+            $results = Infusionsoft_ContactGroupAssignDataService::query(new Infusionsoft_ContactGroupAssign(), array('GroupId' => $id % 10000000, 'ContactId' => floor($id / 10000000)), 1, 0, array('ContactId', 'GroupId', 'DateCreated', 'ContactGroup'), $app);
+            if (count($results) == 0) {
+                throw new Infusionsoft_Exception("Could not load " . $this->table . " with id " . $id);
+            }
+            $object = $results[0];
+
+
+
+            $this->GroupId = $object->GroupId;

```

Comment:
We should be able to use ::load() for this. If we can't, can you add a comment to explain?